spring-cloud-huawei-common/src/main/java/com/huaweicloud/common/transport/ServiceCombSSLProperties.java

```diff
@@ -69,11 +69,11 @@ public void setProject(String project) {
     this.project = project;
   }
 
-  public boolean isEnable() {
-    return enable;
+  public boolean isEnabled() {
+    return enabled;
   }
 
-  public void setEnable(boolean enable) {
-    this.enable = enable;
+  public void setEnabled(boolean enabled) {
```

Comment:
This is a ```CHANGE```，you can do the following
1. read both old and new configuration, and print an warning messesage and suggest users use the new one
2. check if users use the old configuration, and throw and exception and force user use the new one

Reply:
choose the plan2